src/test/regress/expected/pg12.out

```diff
@@ -0,0 +1,90 @@
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 11 AS server_version_above_eleven;
+ server_version_above_eleven
+-----------------------------
+ t
+(1 row)
+
+create schema test_pg12;
+set search_path to test_pg12;
+CREATE FUNCTION blackhole_am_handler(internal)
+RETURNS table_am_handler
+AS 'citus'
+LANGUAGE C;
+CREATE ACCESS METHOD blackhole_am TYPE TABLE HANDLER blackhole_am_handler;
+create table test_am(id int, val int) using blackhole_am;
+insert into test_am values (1, 1);
+-- Custom table access methods should be rejected
+select create_distributed_table('test_am','id');
+ERROR:  only heap AM is supported
+-- Test generated columns
+create table gen1 (
+	id int,
+	val1 int,
+	val2 int GENERATED ALWAYS AS (val1 + 2) STORED
+);
+create table gen2 (
+	id int,
+	val1 int,
+	val2 int GENERATED ALWAYS AS (val1 + 2) STORED
+);
+insert into gen1 (id, val1) values (1,4),(3,6),(5,2),(7,2);
+insert into gen2 (id, val1) values (1,4),(3,6),(5,2),(7,2);
+select * from create_distributed_table('gen1', 'id');
+ERROR:  cannot use column reference in DEFAULT expression
+CONTEXT:  while executing command on localhost:57638
+select * from create_distributed_table('gen2', 'val2');
+ERROR:  cannot use column reference in DEFAULT expression
+CONTEXT:  while executing command on localhost:57638
+insert into gen1 (id, val1) values (2,4),(4,6),(6,2),(8,2);
+insert into gen2 (id, val1) values (2,4),(4,6),(6,2),(8,2);
+select * from gen1;
+ id | val1 | val2 
+----+------+------
+  1 |    4 |    6
+  3 |    6 |    8
+  5 |    2 |    4
+  7 |    2 |    4
+  2 |    4 |    6
+  4 |    6 |    8
+  6 |    2 |    4
+  8 |    2 |    4
+(8 rows)
+
+select * from gen2;
+ id | val1 | val2 
+----+------+------
+  1 |    4 |    6
+  3 |    6 |    8
+  5 |    2 |    4
+  7 |    2 |    4
+  2 |    4 |    6
+  4 |    6 |    8
+  6 |    2 |    4
+  8 |    2 |    4
+(8 rows)
+
+-- Test new VACUUM/ANALYZE options
+analyze (skip_locked) gen1;
+vacuum (skip_locked) gen1;
+vacuum (truncate 0) gen1;
+vacuum (index_cleanup 1) gen1;
+-- COPY FROM
+create table cptest (id int, val int);
+select create_distributed_table('cptest', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+copy cptest from STDIN with csv where val < 4;
```

Comment:
So, the result is wrong. Can't we error out in this PR when COPY has `WHERE` clause?

Reply:
I'll bring #2880 into this PR now that we've decided not to implement support